app/templates/home.html

```diff
@@ -30,10 +30,10 @@ <h1 class="title" align="center">Search Tutorials</h1>
       <img src="{% static 'app/search.png' %}" height="100" width="100">
     </center>
     <br>
-    <form action="{% url 'app:search-results' %}" method="get">
+    <form action="{% url 'app:search-results' %}" method="get" onsubmit="return checkEmpty()">
       <div class="field">
         <div class="control has-icons-left">
-          <input class="input is-rounded" name="q" type="text" placeholder="search tutorials..." id="txt" onkeyup="manage(this)">
+          <input class="input is-rounded" name="q" type="search" placeholder="search tutorials..." id="search-bar">
```

Comment:
the `id` of the search bar is updated but the `checkEmpty()` function is not updated accordingly

Reply:
I have used `checkEmpty()` when the form is submitted. Check the onsubmit event of the form tag.